src/Packagist/Api/Client.php

```diff
@@ -165,9 +165,7 @@ protected function request($url)
 
         return $this->httpClient
             ->get($url)
-            ->send()
-            ->getBody(true)
-        ;
```

Comment:
yep totally! you can ammend your commit (or squash rebase) and then force push. Github will handle it gracefully. thanks :)